components/events/EventCard.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <div class="card">
+        <div class="title_block">
+            <div class="title">{{ title }}</div>
+            <div class="tag" :class="tagClasses()">{{ tag }}</div>
```

Comment:
```suggestion
            <div class="tag" :class="tagClasses">{{ tag }}</div>
```

You can't invoke the computed instance when using it in the template, unlike how we do to a function.
This is the only fix then we can get merged.